assets/js/counter.js

```diff
@@ -0,0 +1,106 @@
+(function ($) {
```

Comment:
For next project : Avoid Jquery (deprecated in recent dev practices) use vanilla js with new functions.

Reply:
Yes